cmd/cli.go

```diff
@@ -99,5 +100,6 @@ func (cli *Cli) RegisterCommands(client *root.Client) {
 		interconnections.NewClient(client, cli.Outputer).NewCommand(),
 		vrf.NewClient(client, cli.Outputer).NewCommand(),
 		virtualcircuit.NewClient(client, cli.Outputer).NewCommand(),
+		vrfroute.NewClient(client, cli.Outputer).NewCommand(),
```

Comment:
Since the route operations are part of the `metal vrf` subcommand, they should be wired up there instead, [similar to how BGP dynamic neighbors is being built](https://github.com/equinix/metal-cli/pull/433/files#diff-68a1cc0c673b754f20006fd3916c6db4120489a84943d561de2a79507ef7f756R59-R62)